src/main/java/frc/robot/subsystems/swerve/module/Modules.java

```diff
@@ -0,0 +1,154 @@
+		for (Module currentModule : modules) {
+			currentModule.setBrake(brake);
+		}
+	}
+
+
+	protected void pointWheels(Rotation2d targetSteerPosition, boolean optimize) {
+		for (Module module : modules) {
+			module.pointSteer(targetSteerPosition, optimize);
+		}
+	}
+
+	protected void pointWheelsInCircle() {
+		boolean optimizeAngle = true;
+		modules[ModuleUtils.ModulePosition.FRONT_LEFT.getIndex()].pointSteer(MathConstants.EIGHTH_CIRCLE.unaryMinus(), optimizeAngle);
+		modules[ModuleUtils.ModulePosition.FRONT_RIGHT.getIndex()].pointSteer(MathConstants.EIGHTH_CIRCLE, optimizeAngle);
+		modules[ModuleUtils.ModulePosition.BACK_LEFT.getIndex()].pointSteer(MathConstants.EIGHTH_CIRCLE, optimizeAngle);
+		modules[ModuleUtils.ModulePosition.BACK_RIGHT.getIndex()].pointSteer(MathConstants.EIGHTH_CIRCLE.unaryMinus(), optimizeAngle);
+	}
+
+	public void pointWheelsInX(boolean isClosedLoop) {
+		SwerveModuleState frontLeftBackRight = new SwerveModuleState(0, MathConstants.EIGHTH_CIRCLE);
+		SwerveModuleState frontRightBackLeft = new SwerveModuleState(0, MathConstants.EIGHTH_CIRCLE.unaryMinus());
+
+		modules[ModuleUtils.ModulePosition.FRONT_LEFT.getIndex()].setTargetState(frontLeftBackRight, isClosedLoop);
+		modules[ModuleUtils.ModulePosition.FRONT_RIGHT.getIndex()].setTargetState(frontRightBackLeft, isClosedLoop);
+		modules[ModuleUtils.ModulePosition.BACK_LEFT.getIndex()].setTargetState(frontRightBackLeft, isClosedLoop);
+		modules[ModuleUtils.ModulePosition.BACK_RIGHT.getIndex()].setTargetState(frontLeftBackRight, isClosedLoop);
+	}
+
+
+	public void stop() {
+		for (Module currentModule : modules) {
+			currentModule.stop();
+		}
+	}
+
+	public void setSteersVoltage(double voltage) {
+		for (Module module : modules) {
+			module.setSteerVoltage(voltage);
+		}
+	}
+
+	public void setDrivesVoltage(double voltage) {
+		for (Module module : modules) {
+			module.setDriveVoltage(voltage);
+		}
+	}
+
+	public void setTargetStates(SwerveModuleState[] moduleStates, boolean isClosedLoop) {
+		for (int i = 0; i < modules.length; i++) {
+			modules[i].setTargetState(moduleStates[i], isClosedLoop);
+		}
+	}
+
+
+	public int getNumberOfOdometrySamples() {
+		int numberOfOdometrySamples = modules[0].getNumberOfOdometrySamples();
+		for (int i = 1; i < modules.length; i++) {
+			numberOfOdometrySamples = Math.min(numberOfOdometrySamples, modules[i].getNumberOfOdometrySamples());
+		}
+		return numberOfOdometrySamples;
+	}
+
+	public SwerveModuleState[] getTargetStates() {
+		return Arrays.stream(modules).map(Module::getTargetState).toArray(SwerveModuleState[]::new);
+	}
+
+	public SwerveModuleState[] getCurrentStates() {
+		return Arrays.stream(modules).map(Module::getCurrentState).toArray(SwerveModuleState[]::new);
+	}
+
+	public Rotation2d[] getDrivesPositions() {
+		return Arrays.stream(modules).map(Module::getDrivePosition).toArray(Rotation2d[]::new);
+	}
+
+	public SwerveModulePosition[] getWheelsPositions(int odometrySampleIndex) {
```

Comment:
Maybe rename from getWheelsPositions()
To
getModulesPositions since its returning the module's position?